app/knight_arena/fight.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from app.peoples.knight_create import KnightCreate
+
+
+class Fight:
+    def __init__(self, first_knight: "KnightCreate",
+                 second_knight: "KnightCreate") -> None:
+        self.first_knight = first_knight
+        self.second_knight = second_knight
+
+    def start_battle(self) -> tuple[Any, Any]:
+        damage_to_first_knight = (
+            max(self.second_knight.power - self.first_knight.protection, 0))
+        self.first_knight.hp -= damage_to_first_knight
+        self.fell_checkout()
+
+        damage_to_second_knight = (
+            max(self.first_knight.power - self.second_knight.protection, 0))
```

Comment:
The `protection` attribute is not directly accessible from the `KnightCreate` class. Ensure that the protection value is calculated and stored in the knight object before the battle starts. You might need to call a method like `apply_protection()` before accessing `self.second_knight.protection`.